Polygon_mesh_processing/test/Polygon_mesh_processing/triangulate_faces_test.cpp

```diff
@@ -90,15 +92,131 @@ test_triangulate_face()
   return true;
 }
 
+template <typename K>
+bool
+test_triangulate_triangle_face()
+{
+  typedef typename K::Point_3                    Point;
+  typedef CGAL::Surface_mesh<Point>          Surface_mesh;
+
+  Surface_mesh mesh;
+  std::ifstream input("data/tetra3.off");
+
+  if (!input || !(input >> mesh) || mesh.is_empty())
+  {
+    std::cerr << "Not a valid off file." << std::endl;
+    return false;
+  }
+
+  BOOST_FOREACH(typename boost::graph_traits<Surface_mesh>::face_descriptor fit, faces(mesh))
+  {
+    if(!CGAL::Polygon_mesh_processing::triangulate_face(fit, mesh))
+      assert(false);
+  }
+  return true;
+}
+
+// todo: add this in Dual.h
```

Comment:
@afabri do you agree that this should go into `CGAL/boost/graph/Dual.h` or is it too specific?

Reply:
ping @afabri

Reply:
ping @afabri